c/include/ml-api-service.h

```diff
@@ -224,9 +224,99 @@ int ml_service_query_create (ml_option_h option, ml_service_h *handle);
 int ml_service_query_request (ml_service_h handle, const ml_tensors_data_h input, ml_tensors_data_h *output);
 
 /**
- * @brief TBU
+ * @brief Registers new information of a neural network model.
+ * @since_tizen 7.5
+ * @remarks Only one model can be activated with given @a name. If same name is already registered in machine learning service, this returns no error and old model will be deactivated when the flag @a activate is true.
+ * @param[in] name The unique name to indicate the model.
+ * @param[in] path The path to neural network model.
+ * @param[in] activate The flag to set the model to be activated.
+ * @param[in] description Nullable, description for neural network model.
+ * @param[out] version The version of registered model.
+ * @return 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Given parameter is invalid.
+ */
+int ml_service_model_register (const char *name, const char *path, const bool activate, const char *description, unsigned int *version);
+
+/**
+ * @brief Updates the description of neural network model with given @a name and @a version.
+ * @since_tizen 7.5
+ * @param[in] name The unique name to indicate the model.
+ * @param[in] version The version of registered model.
+ * @param[in] description The description for neural network model.
+ * @return 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Given parameter is invalid.
+ */
+int ml_service_model_update_description (const char *name, const unsigned int version, const char *description);
+
+/**
+ * @brief Activates a neural network model with given @a name and @a version.
+ * @since_tizen 7.5
+ * @param[in] name The unique name to indicate the model.
+ * @param[in] version The version of registered model.
+ * @return 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Given parameter is invalid.
+ */
+int ml_service_model_activate (const char *name, const unsigned int version);
+
+/**
+ * @brief Gets the information of neural network model with given @a name and @ version.
+ * @since_tizen 7.5
```

Comment:
`* @remarks The @a info should be released using ml_info_destroy().`
remarks tag should be added here and other get_* apis

Reply:
I updated this, plz review the changes again.